gatorgrade/generate/generate.py

```diff
@@ -14,8 +14,39 @@ def create_targeted_paths_list():
     # Add those targeted file paths into a list and returns it.
 
 
-def write_yaml_of_paths_list():  # expected input: A path list
+def write_yaml_of_paths_list(path_names):  # expected input: A path list
     """Write YAML file to create gatorgrade file and set default messages."""
-    # Create a new YAML file with PyYaml in the specific path.
+
+    setup_dict = {
+        "setup": "# add setup commands here\n",
+    }
     # Write the default set up messages in YAML file.
-    # List the file paths in specific format.
+
+    path_names = create_targeted_paths_list(["test"])
+    # Set path_names to the create_targeted_paths_list function with the parameter "test"
```

Comment:
These two lines have no issue. But do you think it's a good idea to put comments before source codes?

Reply:
In this case, I think it would improve readability. Thanks!